lib/net/modules/dns-tools.toit

```diff
@@ -0,0 +1,311 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
```

Comment:
Not sure I like 'tools' so much. I don't think of such helper functions and classes as tools.